src/geoserver-publish-geotiff/index.js

```diff
@@ -58,5 +66,14 @@ const geoserverPublishGeoTiff = async (workerJob, inputs) => {
   workerJob.outputs = [];
 };
 
+/**
+ * Check if the GeoServer is running.
+ *
+ * @returns {Boolean} If the GeoServer is running.
+ */
+const isGeoServerAvailable = async () => {
+  return await grc.about.exists();
+}
+
 // Initialize and start the worker process
-initialize(rabbitHost, rabbitUser, rabbitPass, workerQueue, resultQueue, geoserverPublishGeoTiff);
+initialize(rabbitHost, rabbitUser, rabbitPass, workerQueue, resultQueue, geoserverPublishGeoTiff, isGeoServerAvailable, 10);
```

Comment:
would be great if the interval could be configured from the outside, just like the other variables

Reply:
Well its a number, not a variable, right?

```suggestion
initialize(rabbitHost, rabbitUser, rabbitPass, workerQueue, resultQueue, geoserverPublishGeoTiff, isGeoServerAvailable, connectionInterval);
```
This is what i meant. So of course this would have to be defined on top through process.env, so that i can be given from outside

Reply:
ahh nevermind, was looking at the implementation and not the base class. everything ok!